common/preprocessor.f90

```diff
@@ -15,14 +15,17 @@
 !
 !--------10--------20--------30--------40--------50--------60--------70--------80--------90--------100-------110-------120--------130
 subroutine print_optimize_message
+  use Global_Variables, only: cfunction
   implicit none
   print *, 'Preprocessor: '
-#ifdef ARTED_SC
-  print *, '  ARTED_SC'
-#endif
-#ifdef ARTED_MS
-  print *, '  ARTED_MS'
-#endif
+
+select case(cfunction)
+case ("arted_sc")
+  print *, ' arted_sc'
+case ("arted_ms")
```

Comment:
These lines should be removed since this routine should write only the information of preorocessor macros.